doc/source/conf.py

```diff
@@ -61,8 +61,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'AWS CLI Command Reference'
-copyright = u'2018, Amazon Web Services'
+project = 'AWS CLI Command Reference'
+copyright = f'{datetime.datetime.now().year}, Amazon Web Services'
```

Comment:
We technically don't need a year with this anymore (or to update it at all) but this is fine for now.

Reply:
I did consider removing it, I kind of like some sort of year or timestamp in docs to indicate "freshness", but not a strong opinion if we want to adjust again.